app/src/main/java/com/github/se/cyrcle/model/review/ReviewViewModel.kt

```diff
@@ -10,7 +10,7 @@ import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.update
 
-const val NB_REPORTS_THRESH = 10
+const val NB_REPORTS_THRESH = 1
 const val NB_REPORTS_MAXSEVERITY_THRESH = 3
```

Comment:
The threshold for review is smaller than the one for max severity review i don't understand why maybe you could give me your insight on this ?

Reply:
It's for testing purposes. When we deploy our app in the "real world" NB_REPORTS_THRESH should be set to 10, but if we want the thing to be testable for the coaches and for us, I needed to reduce it. Does that make sense?

Reply:
I understand and you are right maybe I can suggest that you use `NB_REPORTS_MAXSEVERITY_THRESH = min(3,NB_REPORTS_THRESH)` to make it more relevant and robust ? and to keep coherency ?

Reply:
If possible I would then put the const for max severity to 1 just for consistency. Don't break the const convention for this.

Reply:
Works for me